DistributedSystem/Data-Consistency-Model.md

```diff
@@ -0,0 +1,33 @@
+# Data Consistency Model In Distributed System
+
+
+Một trong những thành phần chính tạo ra một hệ thống thông tin, đó là các đối tượng dữ liệu. Các đối tượng dữ liệu trên hệ thống thông tin có thể tồn tại dưới những dạng khác nhau, như các cơ sở dữ liệu, các file hoặc thậm chí là các đối tượng nằm trong bộ nhớ RAM của một process. Tuy nhiên, cách thức sử dụng và quản lý các đối tượng dữ liệu trong một hệ thống phân tán có sự khác biệt với một hệ thống đơn tiến trình truyền thống. Khác với các hệ thống đơn tiến trình truyền thống thường chỉ lưu trữ và quản lý các đối tượng dữ liệu dưới dạng một thực thể duy nhất, trong một hệ thống phân tán, một đối tượng dữ liệu có thể tồn tại dưới dạng nhiều thực thể giống nhau - được gọi là các bản sao (replica) của đối tượng.
```

Comment:
vậy cách dẫn dắt của em có nói tới hệ tập trung không? tại sao lại đập thẳng luôn là "hệ thống đơn tiến trình truyền thống".
một hệ tập trung hoàn toàn có thể đa tiến trình nhé